src/content/tools/pub/publishing.md

```diff
@@ -219,6 +219,9 @@ With this command, `dart pub` performs the following tasks:
 1. Verifies that your package follows the [pubspec format][pubspec] and
    [package layout conventions][pkg-layout].
 
+1. Validates that `git status` is clean (warns against publishing files that
+   are tracked in git, but have uncommitted changes).
```

Comment:
This is a long parenthetical. Consider making it part of the sentence or a new second sentence.

Would something like this work?

```suggestion
1. Validates that `git status` is clean. Warns if
   files that are tracked in git have uncommitted changes.
```